### DISCUSSION TITLE
Initializing with plotfile in PeleLM

### DISCUSSION THREAD

**Discussion by RSuryaNarayan** [AMReX-Combustion/PeleLMeX#341] [Q&A]
Is this feature available in PeleLM like we do `pelec.init_pltfile=plt0000` in PeleC?

**Comment by baperry2** [comment]
Yes, this feature is also [available](https://amrex-combustion.github.io/PeleLMeX/manual/html/LMeXControls.html#io-parameters) in PeleLMeX (not PeleLM, which is no longer actively supported). The syntax is `amr.initDataPlt = plt00010`. By default this expects LMeX plot files, but you can also initialize from a PeleC plot file by adding `amr.initDataPltSource = C`.
